grammars/less.cson

```diff
@@ -277,7 +277,7 @@
     'name': 'keyword.operator.less'
   }
   {
-    'match': '\\s(\\=\\=|\\!\\=|\\<\\=|\\>\\=|\\<|\\>|not|or|and|when)\\s'
+    'match': '\\s(\\=\\=|\\!\\=|\\<\\=|\\>\\=|\\<|\\>|not|or|and|(when not)|when)\\s'
     'name': 'keyword.control.logical.operator'
```

Comment:
Can you change this to only target the 1st capture and rename it to be `keyword.control.logical.operator.less`?